src/utils/Errors.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @flow
+ */
+
+const ERR_ACTION_VALUE_NOT_DEFINED :Error = Error('invalid parameter: action.value is required and must be defined');
+const ERR_ACTION_VALUE_TYPE :Error = Error('invalid parameter: action.value is the incorrect type');
+const ERR_INVALID_ROUTE :Error = Error('invalid route: a route must be a non-empty string that starts with "/"');
+const ERR_INVALID_UUID :Error = Error('invalid uuid');
+const ERR_UNEXPECTED_STATE :Error = Error('unexpected state');
+const ERR_WORKER_SAGA :Error = Error('caught exception in worker saga');
```

Comment:
Don't you need `new Error(...)` here?

Reply:
Nope:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#function_call_or_new_construction

Reply:
No. Calling Error as a function is the same.

http://es5.github.io/#x15.11.1